scraper/src/coordinates_scraper.py

```diff
@@ -0,0 +1,52 @@
+import requests
+import os
+
+def start_processing():
+    data = fetch_data()
+    if (len(data) > 0):
+        transformed_data = transform_data(data)
+
+
+def fetch_data() -> list:
+    data = []
+    url = "https://life-api.coronasafe.network/data/oxygen_v2.json"
+    response = requests.get(url)
+    if ("data" in response.json() and len(response.json()["data"]) > 0):
+        data.extend(response.json()["data"])
+    return data
+
+def transform_data(data):
+    for record in data:
+        if (record):
+            address = ""
+            if "address" in record:
+                address += record["address"]
+            if "pincode" in record:
+                address += record["pincode"]
```

Comment:
It's possible that we have an empty address. In that case PINCODE, city, state India.. (or some such convention) might be better.